test/models/ModelsSpec.scala

```diff
@@ -37,6 +37,11 @@ class ModelsSpec extends SpecBase {
       val result04 = jsonTestData01.setObject(jpath04, jvalue04)
       result04 mustBe a[JsError]
 
+      val result05 = jsonTestData03.setObject(
+          JsPath(List(KeyPathNode("key2")) :+ IdxPathNode(0)),
```

Comment:
Could you please format line 41 and 42 as these are not properly formatted? Could you please check using IntelliJ (if you are using the same)?